vllm/executor/ray_gpu_executor.py

```diff
@@ -36,12 +36,12 @@ def _init_executor(self) -> None:
         # which optimizes the control plane overhead.
         # Run vLLM with VLLM_USE_RAY_COMPILED_DAG=1 to enable it.
         # Currently, this requires USE_RAY_SPMD_WORKER=True.
-        self.use_ray_compiled_dag = envs.VLLM_USE_RAY_COMPILED_DAG
+        self.use_ray_compiled_dag: bool = envs.VLLM_USE_RAY_COMPILED_DAG
         # If the env var is set, then we do not distinguish between the
         # "driver worker" vs other workers. Also, the rank 0 worker will
         # be executed in a remote Ray worker. Currently this requires
         # USE_RAY_COMPILED_DAG=True.
-        self.use_ray_spmd_worker = envs.VLLM_USE_RAY_SPMD_WORKER
+        self.use_ray_spmd_worker: bool = envs.VLLM_USE_RAY_SPMD_WORKER
```

Comment:
Is this necessary? I think the types should be annotated in `envs` already.

Reply:
Yes they are, so I suppose this is not necessary